doc/source/api-reference/qibo.rst

```diff
@@ -247,6 +247,12 @@ Data Encoders
 
 We provide a family of algorithms that encode classical data into quantum circuits.
 
+Computational Basis Encoder
+"""""""""""""""""""""""""""
+
```

Comment:
Maybe add a couple of words about this and the other encoders we already have? I remember I was trying to use them some time ago but I had a hard time understanding how to use them. A small example of how to encode data into circuits might help.